docs/examples/ethane_box_oplsaa.ipynb

```diff
@@ -0,0 +1,69 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mbuild as mb\n",
+    "from mbuild.examples import Ethane\n",
+    "from foyer.tests.utils import get_fn\n",
+    "from foyer import Forcefield"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Approach 1 ###\n",
+    "# Create the chemical topology\n",
+    "ethane_fluid = mb.fill_box(compound=Ethane(), n_compounds=100, box=[2, 2, 2])\n",
+    "# Apply and save the topology\n",
+    "ethane_fluid.save(\"ethane-box.top\", forcefield_files=get_fn(\"oplsaa_alkane.xml\"))\n",
+    "ethane_fluid.save(\"ethane-box.gro\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Approach 2 ###\n",
+    "# Create the chemical topology\n",
+    "ethane_fluid = mb.fill_box(compound=Ethane(), n_compounds=100, box=[2, 2, 2])\n",
+    "# Load the forcefield\n",
+    "opls_alkane = Forcefield(forcefield_files=get_fn(\"oplsaa_alkane.xml\"))\n",
+    "# Apply the forcefield to atom-type\n",
+    "ethane_fluid = opls_alkane.apply(ethane_fluid)\n",
```

Comment:
It might be useful to specify that this is a `ParmEd` object, we don't usually make the important distinction between compounds and structures